src/main/java/org/kpmp/OmicsTypeEnum.java

```diff
@@ -0,0 +1,25 @@
+package org.kpmp;
```

Comment:
This information could also live in the DataTypeEnum, rather than a separate enum so that all of these data points are tied together.